opencti-platform/opencti-graphql/src/python/testing/local_synchronizer.py

```diff
@@ -62,6 +62,9 @@ def _process_message(self, msg):
             logging.info("%s", f"Processing event {msg.id}")
             self.count_number += 1
             data = json.loads(msg.data)
+            type = data["data"]["type"]
+            if type == "internal-relationship":
+                return
```

Comment:
well... this is to avoid to count "internal-relationiship" in raw stream . am I right @SouadHadjiat ?

Reply:
yes, we don't want to synchronize internal relationships

Reply:
we however need them in the stream, because we have a inferrence rule for participates-to relationships (user member of an organization, we infere the relationship with parent organizations)